meteor/app/lib/settings.js

```diff
@@ -18,6 +19,7 @@ mozdef = {
   rootAPI: "https://localhost:8444",
   kibanaURL: "https://localhost:9443/app/kibana#",
   enableBlockIP: true,
-  enableClientAccountCreation: false
+  enableClientAccountCreation: false,
+  authenticationType: "meteor-password"
```

Comment:
enableClientAccountCreation is the toggle for enabling meteor's accounts-password.
authenticationType is for telling us which is set.
Need to determine which type to have set as default.